_data-prepper/pipelines/configuration/processors/otel-trace-raw.md

```diff
@@ -25,8 +25,11 @@ trace_flush_interval | No | Integer | Represents the time interval in seconds to
 
 <!---## Configuration
 
-Content will be added to this section.
+Content will be added to this section.--->
 
 ## Metrics
 
-Content will be added to this section.--->
\ No newline at end of file
+In addition to the metrics from [AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java): <!--- This is incomplete. Add the following? "Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java), the OTel trace raw processor introduces the following custom metrics:"--->
```

Comment:
@graytaylor0 This sentence is incomplete. Is adding the following appropriate here? "Add the following? "Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java), the OTel trace raw processor introduces the following custom metrics:"

Reply:
Yes that would be appropriate here